src/practices/LanguageIndependent/EditorConfigIsPresentPractice.ts

```diff
@@ -31,4 +47,10 @@ export class EditorConfigIsPresentPractice implements IPractice {
 
     return PracticeEvaluationResult.notPracticing;
   }
+
+  async fix(ctx: FixerContext) {
+    await ctx.fileService?.createFile(path.resolve(__dirname, '../../../.editorconfig'), editorConfigTemplate);
```

Comment:
```suggestion
    await ctx.fileService?.createFile(path.resolve(ctx.root.fileInspector!.basePath, '.editorconfig'), editorConfigTemplate);
```
You can use a root basePath from FileInspector. It would be probably better to use a component root path (`ctx.fileInspector!.basePath`).

TBD

Reply:
No, we can not definitely use `fileService` in practices. You have to use Inspectors only.